configure.ac

```diff
@@ -213,6 +238,7 @@ AM_CONDITIONAL([netmap_support], [test x$netmap_support = xyes ])
 AM_CONDITIONAL([PKTIO_DPDK], [test x$pktio_dpdk_support = xyes ])
 AM_CONDITIONAL([HAVE_PCAP], [test $have_pcap = yes])
 AM_CONDITIONAL([SDK_INSTALL_PATH_], [test "x${SDK_INSTALL_PATH_}" = "x1"])
+AM_CONDITIONAL([DPDK_DEFAULT_DIR], [test "x${DPDK_DEFAULT_DIR}" = "x1"])
```

Comment:
This PR is mainly about bringing linux-dpdk into cloud & be able to build. Not restructuring of linux-generic & linux-dpdk. I would like consider doing these improvements outside this PR.